### PR TITLE
Feature/profile contacts

### LINKED ISSUE #121: Add the ability to add your phone number
Use the existing update profile mutation.

Also find a way to check if the user has Telegram and/or WhatsApp and flag it somehow

### PR DESCRIPTION
**Pull request by lucca65**
## What issue does this PR close
Closes #121 

## Changes Proposed ( a list of new changes introduced by this PR)
- Add contacts API to add data to profile

### CODE REVIEW

File: lib/cambiatus/error.ex [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    |> Ecto.Changeset.traverse_errors(fn {err, opts} ->
+      Enum.reduce(opts, err, fn {key, value}, acc ->
+        String.replace(acc, "%{#{key}}", to_string(value))
+      end)
+    end)
+  end
+
+  def from(other) do
+    Logger.error("Unhandled error term: \n#{inspect(other)}")
+    Sentry.capture_message("Sign up failed", extra: %{error: other})
```
Comment (amiskov) [inline lib/cambiatus/error.ex:31]: Looks like this line should be more general than `"Sign up failed"`. Maybe like this?

```suggestion
    Sentry.capture_message("Unhandled error term:", extra: %{error: other})
```

File: lib/cambiatus/error.ex [hunk 2]
```diff
@@ -0,0 +1,35 @@
+defmodule Cambiatus.Error do
+  @moduledoc """
+  Standard error structure to be used in Cambiatus.
+
+  Used to represent and display operation statuses in a more standard way
+
+  Errors have two fields:
+  * `type`: atom representing the possible error types
+  * `message`: strings with the error message
+  """
```
Comment (amiskov) [inline lib/cambiatus/error.ex:10]: Wow! Looks very promising 👍

File: lib/cambiatus_web/schema/account_types.ex [hunk 3]
```diff
@@ -3,73 +3,92 @@ defmodule CambiatusWeb.Schema.AccountTypes do
...
+      )
+
+      arg(:invitation_id, :string,
+        description: "Optinal, used to auto invite an user to a community"
+      )
+
+      arg(:kyc, :kyc_data_update_input, description: "KYC data")
+      arg(:address, :address_update_input, description: "Address data")
+
+      resolve(&AccountsResolver.sign_up/3)
```
Comment (amiskov) [inline lib/cambiatus_web/schema/account_types.ex:57]: Really nice to have those description fields!

**Review by amiskov — CHANGES_REQUESTED** [review]
I really like these changes!
While checking updates on the GraphQl schema I tried to generate types on the frontend with `elm-graphql` CLI and it froze. Then I tried to check `http://localhost:4000/api/graphiql` and it sad that there's no schema available. @lucca65 can you please check this wired thing on your side?

File: lib/cambiatus/error.ex (hunk 1 above)
Comment (lucca65) [reply lib/cambiatus/error.ex]: ops, thanks for noticing bro 😀

**Comment by lucca65** [comment]
Found it! turns out we [cannot use objects as input](https://github.com/graphql/graphql-js/issues/151)! Thanks for testing this out before merge @amiskov

**Review by amiskov — APPROVED** [review]
Looks "todo bem"! 👍

**Pull request merged**